include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h

```diff
@@ -962,13 +1007,16 @@ namespace internal
             for (unsigned int i = 0, k = 0; i < ids.size(); ++i)
               {
                 const unsigned int active_fe_index = buffer[k++];
+                const unsigned int refinement_case = buffer[k++];
```

Comment:
Similar to my comment above, do you want to make this reflect the type intended for `refinement_case`, i.e., `std::unit8_t`?

Reply:
The compiler complained when casting straight to `std::unit8_t`, but calling `RefinementCase()` first also seems to fix it.